frontend/src/Store/Actions/movieFileActions.js

```diff
@@ -168,15 +169,12 @@ export const actionHandlers = handleThunks({
       dispatch(batchActions([
         ...movieFileIds.map((id) => {
           const props = {};
-
-          if (language) {
-            props.language = language;
+          if (languages) {
+            props.languages = languages;
           }
-
           if (quality) {
             props.quality = quality;
           }
-
```

Comment:
Keep the lines between statements for readability.... these get minimized during build so it won’t affect anything.

Reply:
Add back this newline